.github/workflows/dependabot-batcher.yml

```diff
@@ -6,10 +6,10 @@ name: 'Dependabot Batcher for Paella 7'
 on:
   pull_request:
     paths:
-      - '/modules/engage-paella-player-7/**'
+      - 'modules/engage-paella-player-7/**'
```

Comment:
This should always fail since you cannot access write secrets when acting on a `pull_request` trigger. This makes sense since the workflow is executed on whatever any user changes. Allowing something like that would mean that someone could just modify this workflow in a pull request and e.g. delete all of Opencast's code (or do other nasty things).

But you don't actually need `pull_request` at all, since whenever Dependabot updates a pull request, it will also push code to the repository. That's why push should be sufficient.

Reply:
That's right, I was yesterday talking about it in Matrix over this problem. When you manually execute the action, the token gets write-all permissions, but not when is automatically triggered.

Fortunately, there is a fix: I can add the permissions' parameter in with I can add the writing rights to the token. Still, I haven't found what right is the correct one to enable. (I'm not comfortable in enable writing rights to all scopes)

Reply:
Solved!